website/src/pages/evaluate/rank_initial_prompts.tsx

```diff
@@ -0,0 +1,144 @@
+import { ArrowUpIcon, ArrowDownIcon } from "@heroicons/react/20/solid";
+import Head from "next/head";
+import { useState } from "react";
+import useSWRImmutable from "swr/immutable";
+import useSWRMutation from "swr/mutation";
+import clsx from "clsx";
```

Comment:
Could this be ordered above alphabetically?

Reply:
gladly.

might be worth adding an eslint rule:

https://eslint.org/docs/latest/rules/sort-imports

should I create a new issue for this?

Reply:
yes please~